packages/samples/atomic/package.json

```diff
@@ -9,6 +9,9 @@
     "url": "https://github.com/codesandbox-app/static-template/issues"
   },
   "homepage": "https://github.com/codesandbox-app/static-template#readme",
+  "scripts": {
+    "prepublishOnly": "exit 1"
```

Comment:
This script was copied from `headless-react`'s scripts. It blocks `npm publish`.